solvis/module.php

```diff
@@ -109,20 +109,27 @@ public function ApplyChanges()
 
 				/* ****** Solvis Register ************************************************************************** */
 				$modelRegister_array = array(
-					array(2049,"R","Zirkulation Betriebsart", "Zirkulation: 1 - Aus 2 - Puls 3 - Temp 4 - Warten","int16", "enumerated_Zirkulation"),
-					array(3840,"R","Analog Out 1", "Betriebsart: Status,0 - Auto PWM 1 - Hand PWM 2 - Auto analog 3 - Hand analog","int16", ""),
-					array(3845,"R","Analog Out 2", "Betriebsart: Status,0 - Auto PWM 1 - Hand PWM 2 - Auto analog 3 - Hand analog","int16", ""),
-					array(3850,"R","Analog Out 3", "Betriebsart: Status,0 - Auto PWM 1 - Hand PWM 2 - Auto analog 3 - Hand analog","int16", ""),
-					array(3855,"R","Analog Out 4", "Betriebsart: Status,0 - Auto PWM 1 - Hand PWM 2 - Auto analog 3 - Hand analog","int16", ""),
-					array(3860,"R","Analog Out 5", "Betriebsart: Status,0 - Auto PWM 1 - Hand PWM 2 - Auto analog 3 - Hand analog","int16", ""),
-					array(3865,"R","Analog Out 6", "Betriebsart: Status,0 - Auto PWM 1 - Hand PWM 2 - Auto analog 3 - Hand analog","int16", ""),
+					//Solvis.Zirkulation
```

Comment:
Wofür möchtest du das Profil hier kommentieren?
Das steht doch schon in der Description und ist in IPS somit einsehbar:
![image](https://user-images.githubusercontent.com/39644775/136245297-b66da306-6e2a-4b40-9de9-5521256c5d32.png)